frontend/dist/assets/tracim/appInterface.js

```diff
@@ -27,6 +27,8 @@ var getSelectedApp = function (appName) {
         return (appWorkspaceAdvanced || {default: {}}).default
       case 'agenda':
         return (appAgenda || {default: {}}).default
+      case 'office_document':
```

Comment:
rename to collaborative-document-edition

Reply:
At the moment the only purpose of this app is to create office document, it doesn't edit them.
Do you still want a rename ?